.github/workflows/lint.yaml

```diff
@@ -0,0 +1,21 @@
+name: ci
+on:
+  push:
+    branches:
+      - main
+  pull_request:
+    paths-ignore:
+    - '**/*.md'
+jobs:
+  unit:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Check out repository code
+        uses: actions/checkout@v2
+      - name: Install Go 1.17
+        uses: actions/setup-go@v2
+        with:
+          go-version: '1.17'
+      - name: Run unit tests
```

Comment:
Copy/paste error: you'll want something like "Run sanity checks", or something along those lines.